From 13fed80854c281cc7eaf78e5848b10f5becb947a Mon Sep 17 00:00:00 2001
From: Jooyung Han <jooyung@google.com>
Date: Tue, 26 Jul 2022 12:56:00 +0900
Subject: [PATCH 1/3] Add netlink public header library

libwifi-hal-bcm has been using the header via LOCAL_C_INCLUDES. To
convert libwifi-hal-bcm into a Soong module, the header is exposed as a
header lib.

Bug: 239984067
Test: m libwifi-hal-bcm
Merged-In: I92e044565b291c003cff9ac580dd36299fab9851
Change-Id: I92e044565b291c003cff9ac580dd36299fab9851
(cherry picked from commit eb043d58c236676a019ff8579d46b06f83a6daa1)
---
 Android.bp | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/Android.bp b/Android.bp
index cbb1a90f..35cdb4f5 100644
--- a/Android.bp
+++ b/Android.bp
@@ -26,3 +26,11 @@ license {
     ],
     license_text: ["LICENSE"],
 }
+
+cc_library_headers {
+    name: "libnetlink_iface_headers",
+    vendor_available: true,
+    export_include_dirs: [
+        "src/drivers",
+    ],
+}
-- 
2.39.2

